docs/3-index-pages.md

```diff
@@ -218,7 +218,7 @@ You can change it per request / action too:
 
 ```ruby
 controller do
-  before_filter :only => :index do
+  before_action only: :index do
```

Comment:
No, `before_action` only works with Rails >= 4

This is just the documentation. I think it's fine to exclude Rails 3.2 from the documentation, but feel free to update it if you'd rather the documentation be more comprehensive.